### PR TITLE
Remove -base suffix from color token names

### LINKED ISSUE #21: Update color names 🎨
🎨 Colors were updated on <https://shopify.invisionapp.com/dsm/shopify/design-tokens>, please update this repository!

### Changes

Updated Polaris color names in InVision DSM. https://github.com/Shopify/polaris-tokens/pull/22

### Follow these steps

* [x] If you haven’t yet, clone this repository
  ```
  dev clone polaris-tokens
  dev up
  ```
* [x] Fetch tokens from Invision
  ```
  dev cd polaris-tokens
  git checkout master
  git pull
  git checkout -B update-color-tokens
  yarn getTokensFromInvision
  yarn dist
  yarn test -u
  ```
* [x] Update `CHANGELOG.md`
* [x] Commit & push the changes
  ```
  git commit -am "Update color tokens to the latest version"
  git push origin update-color-tokens
  ```
* [x] [Open a pull request][pr-link]

[pr-link]: https://github.com/Shopify/polaris-tokens/compare/master...update-color-tokens?expand=1&title=Update%20color%20tokens&body=Fixes%20%23XXX%20%3C!--%20replace%20XXX%20with%20the%20id%20of%20the%20original%20issue%20--%3E&assignees=kaelig,amrocha,tmlayton

### PR DESCRIPTION
**Pull request by sarahill**
Updated color names in InVision DSM.

Fixes #21

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+## [1.3.0] - 2018-06-13
+
+* Updated color names in InVision DSM
```
Comment (kaelig) [inline CHANGELOG.md]: Let's mention it fixes #16 and it is a breaking change so we'll need to bump up to `2.0.0`.

File: dist/colors.color-map.scss [hunk 2]
```diff
@@ -1,90 +1,79 @@
...
   'green':
     (
-      'lighter': rgb(227, 241, 223),
-      'light': rgb(187, 229, 179),
-      'base': rgb(80, 184, 60),
-      'dark': rgb(16, 128, 67),
+      'text': rgb(65, 79, 62),
       'darker': rgb(23, 54, 48),
-      'text': rgb(65, 79, 62)
+      'dark': rgb(16, 128, 67),
```
Comment (kaelig) [inline dist/colors.color-map.scss]: `base` colors are missing in this file - the `getShade` function needs to be adapted for this use case.

**Comment by kaelig** [comment]
Thanks for taking this one! I commented on a couple of issues - should be an easy fix.

File: dist/colors.color-map.scss [hunk 3]
```diff
@@ -1,90 +1,79 @@
 $polaris-colors: (
```
Comment (kaelig) [inline dist/colors.color-map.scss]: Black and white are missing. Let me know if you need help - happy to pair on resolving it.

Comment (sarahill) [reply dist/colors.color-map.scss]: Yea, pairing on this would be great. Thought I'd resolved the issue but things keep popping up missing :p

**Comment by sarahill** [comment]
@kaelig got all the colors updated in the Web UI Kit and in DSM.

File: dist/colors.color-map.scss [hunk 4]
```diff
@@ -1,84 +1,84 @@
...
-    'light': rgb(227, 208, 255),
-    'base': rgb(156, 106, 222),
-    'dark': rgb(80, 36, 143),
-    'darker': rgb(35, 0, 81),
     'text': rgb(80, 73, 90),
+    'darker': rgb(35, 0, 81),
+    'dark': rgb(80, 36, 143),
+    'base': rgb(156, 106, 222),
+    'light': rgb(227, 208, 255),
+    'lighter': rgb(245, 240, 252),
```
Comment (kaelig) [inline dist/colors.color-map.scss]: purple-lighter has changed, is that normal?

Before / after:

```diff
- rgb(246, 240, 253)
+ rgb(245, 240, 252)
```

File: dist/colors.color-map.scss [hunk 5]
```diff
@@ -1,84 +1,84 @@
...
     'dark': rgb(196, 205, 213),
-  ),
-  'ink': (
-    'lightest': rgb(145, 158, 171),
-    'lighter': rgb(99, 115, 129),
-    'light': rgb(69, 79, 91),
-    'base': rgb(33, 43, 54),
+    'base': rgb(223, 227, 232),
+    'light': rgb(244, 246, 248),
+    'lighter': rgb(249, 250, 251),
```
Comment (kaelig) [inline dist/colors.color-map.scss]: ink / lightest is missing

File: dist/index.d.ts [hunk 6]
```diff
@@ -1,61 +1,61 @@
...
-  colorBlueBase: "rgb(0, 122, 206)";
+  colorBlack: "rgb(0, 0, 0)";
+  colorBlue: "rgb(0, 122, 206)";
   colorBlueDark: "rgb(8, 78, 138)";
-  colorBlueDarker: "rgb(0, 20, 41)";
-  colorBlueLight: "rgb(180, 225, 250)";
-  colorBlueLighter: "rgb(235, 245, 250)";
+  colorBlueDarker: "rgb(0, 21, 42)";
+  colorBlueLight: "rgb(180, 224, 250)";
+  colorBlueLighter: "rgb(235, 244, 250)";
```
Comment (kaelig) [inline dist/index.d.ts]: Blue shades seem to have changed, is that on purpose?

File: dist/index.d.ts [hunk 7]
```diff
@@ -1,61 +1,61 @@
...
   colorGreenLight: "rgb(187, 229, 179)";
   colorGreenLighter: "rgb(227, 241, 223)";
   colorGreenText: "rgb(65, 79, 62)";
-  colorIndigoBase: "rgb(92, 106, 196)";
+  colorIndigo: "rgb(92, 106, 196)";
   colorIndigoDark: "rgb(32, 46, 120)";
   colorIndigoDarker: "rgb(0, 6, 57)";
   colorIndigoLight: "rgb(179, 188, 245)";
-  colorIndigoLighter: "rgb(244, 245, 250)";
+  colorIndigoLighter: "rgb(243, 244, 250)";
```
Comment (kaelig) [inline dist/index.d.ts]: colorIndigoLighter has also slightly changed

**Comment by kaelig** [comment]
I had to rebase to update tests and perform a new review.

Here's how to get the rebased branch locally:

```
git fetch
git reset origin/update-dsm-color-names --hard
```

File: dist/colors.color-map.scss (hunk 4 above)
Comment (sarahill) [reply dist/colors.color-map.scss]: When we looked at the diff together there were values that had changed so I updated them in this round back to the previous values that were in this repo. So they have changed but it's just correcting them back to what they were.

Comment (kaelig) [reply dist/colors.color-map.scss]: Some colors are still off. This also means that people using DSM (if there are any) are using the wrong colors at this time.

Here's the baseline that DSM should follow: https://shopify.github.io/polaris-tokens/

Hope this helps!

**Comment by sarahill** [comment]
Ok that page helped a lot. Think I got it 🤞

**Comment by kaelig** [comment]
Thanks @sarahill! This is merged to master - we'll cut a v2.0.0 release very soon :)

**Pull request merged**